test/tuned_models.jl

```diff
@@ -406,4 +406,53 @@ end
     MLJBase.Tables.istable(mach.cache[end].fitresult.machine.data[1])
 end
 
+# define a supervised model with ephemeral `fitresult`, but which overcomes this by
+# overloading `save`/`restore`:
+thing = []
+struct EphemeralRegressor <: Deterministic end
+function MLJBase.fit(::EphemeralRegressor, verbosity, X, y)
+    # if I serialize/deserialized `thing` then `view` below changes:
+    view = objectid(thing)
+    fitresult = (thing, view, mean(y))
+    return fitresult, nothing, NamedTuple()
+end
+function MLJBase.predict(::EphemeralRegressor, fitresult, X)
+    thing, view, μ = fitresult
```

Comment:
I think we should rename the `view` variable name.

Reply:
done